CFPDXDataStorageObject.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
```

Comment:
This sort of file header is used when a file is run as a script from the command line.  It isn't necessary when a module is used from within a Python app.